creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -0,0 +1,46 @@
+const prometheusClient = require('prom-client')
```

Comment:
nit - can we capitalize this package name? otherwise looks a lot like variable name / instance name

Reply:
yep